src/init.cpp

```diff
@@ -1235,6 +1240,8 @@ bool AppInit2(boost::thread_group& threadGroup, CScheduler& scheduler)
         return false;
 #endif // ENABLE_WALLET
 
+    fPreferV4Transactions = GetBoolArg("-preferv4transactions", DEFAULT_SEND_FREE_TRANSACTIONS);
```

Comment:
Why is the default for this `DEFAULT_SEND_FREE_TRANSACTIONS`?

Also, this should probably be an integer flag instead (with a range of acceptable values that we update as we change the transaction builder), so that we can keep this meaningful going forward instead of it quickly becoming vestigial:
```suggestion
    nPreferredTxVersion = GetArg("-prefertxversion", 5);
```